### DISCUSSION TITLE
Can multiple 'Extra' groups be specified with -g flag ?

### DISCUSSION THREAD

**Discussion by edschreibman** [prowler-cloud/prowler#1462] [Q&A] [answered]
Can multiple 'Extra' groups be specified with -g flag ?
I.E. similar to how multiple checks can be specified with -c:   ./prowler -c check310,check722
Example:      ./prowler -g gdpr,forensics-ready,hipaa

**Comment by edschreibman** [comment]
Tested, Does not appear so

**Comment by jfagoagas** [comment, marked as answer]
Hi @edschreibman, you can do it using this way: `./prowler -p dev -g gdpr group1`

Thanks for using Prowler!
